unreachable code

<!-- Please describe your issue in English. -->

#### Can be reproduced ?
Yes

#### Steps:
go vet ./...

#### Expected behavior:
OK

#### Actual behavior:
components/prophet/schedulers/base_scheduler.go:66:2: unreachable code

#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:

#### Configuration file:


#### Additional context:
- Error message from client:
- Server log:
- Other information:

--- a/components/prophet/schedulers/base_scheduler.go
+++ b/components/prophet/schedulers/base_scheduler.go
@@ -63,7 +63,6 @@ func intervalGrow(x time.Duration, maxInterval time.Duration, typ intervalGrowth
 	default:
 		panic(fmt.Sprintf("type %+v error", typ))
 	}
-	return 0
 }
 
 // BaseScheduler is a basic scheduler for all other complex scheduler

